ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -913,7 +913,7 @@
       },
       "outputs": [],
       "source": [
-        "if os.getenv(\"IS_TESTING\") is True:\n",
+        "if os.getenv(\"IS_TESTING\") == \"1\":\n",
```

Comment:
Feel free to use

```
if os.getenv(\"IS_TESTING\"):
```

Which is less explicit but is more standardized.

Reply:
Done.